velox/functions/sparksql/String.h

```diff
@@ -1181,6 +1181,35 @@ struct FindInSetFunction {
   }
 };
 
+template <typename T>
+struct RepeatFunction {
```

Comment:
nit: add some comments for this function. Some example in the replace function:

> /// replace(input, replaced) -> varchar
///
///     Removes all instances of ``replaced`` from ``input``.
///     If ``replaced`` is an empty string, returns the original ``input``
///     string.

Reply:
added, thanks